src/video/win32_v.cpp

```diff
@@ -993,6 +993,25 @@ void VideoDriver_Win32Base::EditBoxLostFocus()
 	SetCandidatePos(this->main_wnd);
 }
 
+std::vector<int> VideoDriver_Win32Base::GetListOfMonitorRefreshRates()
+{
+	std::vector<int> rates = {};
+	EnumDisplayMonitors(nullptr, nullptr, [](HMONITOR hMonitor, HDC hDC, LPRECT rc, LPARAM data) -> BOOL {
+		auto& list = *reinterpret_cast<std::vector<int>*>(data);
+		MONITORINFOEX monitorInfo = {};
+		monitorInfo.cbSize = sizeof(MONITORINFOEX);
+		GetMonitorInfo(hMonitor, &monitorInfo);
+		DEVMODE devMode = {};
```

Comment:
I think this lambda becomes a lot more readable if you add a newline between every command. So:

```
GetMonitorInfo(hMonitor, &monitorInfo);

DEVMODE devMode = {};
```

etc. It now really is just lines of code that are very hard to process :D